support-frontend/window.d.ts

```diff
@@ -8,25 +9,28 @@ declare global {
 	 */
 	interface Window {
 		guardian: {
```

Comment:
the diff is a little hard to read as I've alphabetised the properties, here I'd added `user`, `email` and `uatmode`